docs/components/concepts/document-handling.md

```diff
@@ -5,56 +5,85 @@ description: "Learn more about integration, secure management, and efficient sto
-- [Amazon S3](/components/connectors/out-of-the-box-connectors/amazon-s3.md) - Supports uploading documents from (or downloading documents to) the Camunda document store.
-- [Box](/components/connectors/out-of-the-box-connectors/box.md) - Supports uploading documents from (or downloading documents to) the Camunda document store.
+### Upload a document via public form
 
-![example REST configuration](./assets/rest-outbound-document.png)
+A [form](/components/modeler/forms/camunda-forms-reference.md) can display documents with the [document preview component](/components/modeler/forms/form-element-library/forms-element-library-document-preview.md):
+
+![document preview settings](./assets/document-preview-settings.png)
+
+With the [filepicker form element](/components/modeler/forms/form-element-library/forms-element-library-filepicker.md), you may also [upload multiple files at once](/components/modeler/forms/form-element-library/forms-element-library-filepicker.md#configurable-properties). This can be dynamically set using an [expression](/components/modeler/feel/language-guide/feel-expressions-introduction.md):
+
+![document preview for form](./assets/document-preview.png)
 
-## Tasklist
+For additional guidance on supported file formats, refer to the [MDN Web Docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/file#unique_file_type_specifiers).
 
-Tasklist users may view and download files displayed in the task's form.
+With Tasklist, users may then view and download files displayed in the task's form.
 
 :::note
 If you change the **Document URL** under **Download settings**, this feature will not work for public start forms because there is no valid session.
 :::
 
-![document preview settings](./assets/document-preview-settings.png)
+![document handling in tasklist](./assets/document-handling-tasklist.png)
 
-## Document limitations
+#### Outbound Connectors
 
-Document handling maintains the following guidelines:
+The [Connector SDK](/components/connectors/custom-built-connectors/connector-sdk.md) is enhanced to provide document support in property/variable bindings.
 
-- **Maximum file size**: 10 MB per file.
-- **File expiration time/time-to-live (TTL) policy**: 30 days by default. Clients for Connectors and Forms may specify a custom expiration date when uploading documents.
+For the following outbound Connectors, you may include a **Request body** under **Payload** in the properties panel to send with your request:
+
+- [REST](/components/connectors/protocol/rest.md) - Supports storing the response as a document.
+- [Email](/components/connectors/out-of-the-box-connectors/email.md#response-structure-1) - Supports sending Camunda documents as attachments, or storing incoming attachments as Camunda documents.
+- [Amazon Textract](/components/connectors/out-of-the-box-connectors/amazon-textract.md) - Can read the input document directly from the Camunda document store.
```

Comment:
@chillleader Can you direct me to the UI elements that must be adjusted for Amazon Textract, Bedrock, S3, and Box? Alternatively I can put some time on your calendar to discuss.

Reply:
Sure, in each of these connectors there will be a field called "Document" where the document reference can be provided. You can already see this with the S3 connector, for example, for others it will be similar

<img width="384" alt="Screenshot 2025-02-17 at 16 05 58" src="https://github.com/user-attachments/assets/7a6182a2-7d96-4d97-86dc-3d4660ed740d" />